### DISCUSSION TITLE
Import from Gitlab is failing

### DISCUSSION THREAD

**Discussion by gokumar** [community/community#35396] [Other Feature Feedback, Questions, & Ideas]
Hi,

I am trying to migrate my code base from Gitlab to Github. Till early this week, I was able import some of the repositories, but from today, Import tool is failing. Please find the error.

![Screenshot_20221007_105914](https://user-images.githubusercontent.com/14192886/194474454-940fa9aa-b359-43f3-a154-52fee31c7aa1.png)

I tried using "Restart" option, but does not help. I got this error for 5 out of 6 repositories that I tried importing today.

**Comment by jsoref** [comment]
Time to contact support
